multichain-testing/src/register-interchain-bank-assets.builder.js

```diff
@@ -0,0 +1,51 @@
+/* global harden */
+/// <reference types="ses" />
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {ParseArgsConfig} from 'node:util';
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js';
+ */
+
+/** @type {ParseArgsConfig['options']} */
+const parserOpts = {
+  assets: { type: 'string' },
+};
+
+/** @type {CoreEvalBuilder} */
+export const defaultProposalBuilder = async (_, options) => {
+  return harden({
+    sourceSpec:
+      '@agoric/builders/scripts/testing/register-interchain-bank-assets.js',
+    getManifestCall: ['getManifestCall', options],
+  });
+};
+
+/** @type {DeployScriptFunction} */
+export default async (homeP, endowments) => {
+  const { scriptArgs } = endowments;
+
+  const {
+    values: { assets },
+  } = parseArgs({
+    args: scriptArgs,
+    options: parserOpts,
+  });
+
+  const parseAssets = () => {
+    if (typeof assets !== 'string') {
+      throw Error(
+        'must provide --assets=JSON.stringify({ denom: Denom; keyword: string; decimalPlaces: number; }[])',
```

Comment:
Is this really a keyword? Or is it a misnomer where we really mean `issuerName`?

```suggestion
        'must provide --assets=JSON.stringify({ denom: Denom; issuerName: string; decimalPlaces: number; }[])',
```

Reply:
I remember this comment in the add-vault proposal. I'll look to make the corresponding change in the builder so we don't continue to carry the tech debt